test/plugin.html-validate.external-links.mjs

```diff
@@ -51,7 +71,7 @@ export default class ExternalLinksRule extends Rule {
   setup() {
     this.db = this.setupDatabase();
     this.skipUrlsRegex = this.loadSkipUrls();
-    this.on("tag:ready", this.tagReady.bind(this));
+    this.on("dom:ready", this.domReady.bind(this));
```

Comment:
Please see https://html-validate.org/dev/events.html#dom-nodes

`tag:ready` should be what we want